airbyte-integrations/connectors/source-tweag-blog/source_tweag_blog/source.py

```diff
@@ -206,17 +220,9 @@ def stop_gatsby_server(self, gatsby_process) -> None:
             gatsby_process.kill()
         self.free_port(12123)
 
-    def check_connection(self, logger, config) -> Tuple[bool, any]:
+    def check_connection(self) -> Tuple[bool, any]:
         """Check if the source is reachable"""
-        try:
-            self.clone_repo(config["repo_url"], "/tmp/repo")
-            gatsby_process = self.start_gatsby_server("/tmp/repo")
-            time.sleep(30)
-            self.stop_gatsby_server(gatsby_process)
-            return True, None
-        except Exception as e:
-            logger.error(f"Connection check failed: {str(e)}")
-            return False, str(e)
+        return True, None
```

Comment:
I first wondered whether this is in line with what's intended from `check_connection()`. But it seems that this function really only serves to confirm basic connectivity between Airbyte and this source objects, e.g. correct credentials. But stream readiness can be checked on the stream level, e.g. in the `read_records()` function.